pokemongo_bot/cell_workers/spin_fort.py

```diff
@@ -100,8 +105,9 @@ def work(self):
                 result = c.fetchone()
                 c.execute("SELECT DISTINCT COUNT(pokestop) FROM pokestop_log WHERE dated >= datetime('now','-1 day')")
                 if c.fetchone()[0] >= self.config.get('daily_spin_limit', 2000):
-                    self.emit_event('spin_limit', formatted='WARNING! You have reached your daily spin limit')
-                    sys.exit(2)
+                   if datetime.now() >= self.next_update:
+                       self.emit_event('spin_limit', formatted='WARNING! You have reached your daily spin limit')
+                       self._compute_next_update()
```

Comment:
I added a new variable called 'bypass_daily_limit'. This is "false" by default and if the daily_spin_limit is reached, the code will exit. This is the current behavior.

But if the 'bypass_daily_limit' is set to true and the daily_spin_limit is reached, the bot will continue its business but it will give a warning message that the daily spin limit has been reached every 120 seconds (through the min_interval variable which can be changed).